habitat-lab/habitat/gui/gui_application.py

```diff
@@ -0,0 +1,221 @@
+#!/usr/bin/env python3
```

Comment:
Everything under `habitat-lab/habitat/gui` should be moved to `examples/siro_sandbox/gui` or similar. We don't want any sandbox-specific code in the habitat-lab library. See https://app.asana.com/0/1205278804336687/1205096464766198 for details.